isis/src/lro/apps/lronaccal/lronaccal.cpp

```diff
@@ -0,0 +1,938 @@
+/** This is free and unencumbered software released into the public domain.
+
+The authors of ISIS do not claim copyright on the contents of this file.
+For more details about the LICENSE terms and the AUTHORS, you will
+find files of those names at the top level of this repository. **/
+
+/* SPDX-License-Identifier: CC0-1.0 */
+
+#include "ProcessByLine.h"
+#include "SpecialPixel.h"
+#include "Message.h"
+#include "Camera.h"
+#include "iTime.h"
+#include "IException.h"
+#include "TextFile.h"
+#include "Brick.h"
+#include "Table.h"
+#include "PvlGroup.h"
+#include "Statistics.h"
+#include "UserInterface.h"
+#include "lronaccal.h"
+#include <fstream>
+#include <QTextStream>
+#include <QDir>
+#include <QRegExp>
+#include <QString>
+#include <vector>
+
+using namespace std;
+namespace Isis {
+
+  // Working functions and parameters
+  void ResetGlobals();
+  void CopyCubeIntoVector(QString &fileString, vector<double> &data);
+  void ReadTextDataFile(QString &fileString, vector<double> &data);
+  void ReadTextDataFile(QString &fileString, vector<vector<double> > &data);
+  void Calibrate(Buffer &in, Buffer &out);
+  void RemoveMaskedOffset(Buffer &line);
+  void CorrectDark(Buffer &in);
+  void CorrectNonlinearity(Buffer &in);
+  void CorrectFlatfield(Buffer &in);
+  void RadiometricCalibration(Buffer &in);
+  void GetNearestDarkFile(QString fileString, QString &file);
+  void GetNearestDarkFilePair(QString &fileString, QString &file0, QString &file1);
+  void GetCalibrationDirectory(QString calibrationType, QString &calibrationDirectory);
+  void GetWeightedDarkAverages();
+  bool AllowedSpecialPixelType(double pixelValue);
```

Comment:
Might be worth making these static, since they are added to libisis they risk potentially overlapping with something else in the ISIS namespace.